deploy/Jenkinsfile

```diff
@@ -356,4 +356,16 @@ pipeline {
             }
         }
     }
+
+    post {
+        changed {
+            script {
+                if (!env.ghprbPullId) {
+                    mail to: 'bbp-deployment@groupes.epfl.ch',
+                         subject: "Failed Deployment Build: ${currentBuild.fullDisplayName}",
+                         body: "See ${env.BUILD_URL}"
+                }
+            }
+        }
```

Comment:
I assume this is only for nightly builds? Hope not too many emails :) Lets see!

Reply:
Yes. Actually just noticed that I forgot a condition there / this should be for failed builds only

Reply:
I think it's good to make this for every failed build, to really drive home when it fails continuously. The volume should still be much less than these permanent "issue has been moved to next sprint" notifications we receive.

Reply:
totally agree on the sprint notifications